panoramisk/fast_agi.py

```diff
@@ -46,16 +46,16 @@ def call_waiting(request):
         return agi_result
 
 
-class Application(dict):
```

Comment:
I found that Application could easily clash as it's a very generic name, but I am happy to revoke it as it can simply be solved with : "from panoramisk.fast_agi import Application as FastAGI_App"